app/web/src/newhotness/Explore.vue

```diff
@@ -852,20 +837,22 @@ watch([focusedComponentIdx], () => {
   }
 });
 
-const focusedGridComponent = computed(() =>
+const focusedGridComponentRef = computed(() =>
   focusedComponentIdx.value > -1
-    ? exploreGridRef.value?.getGridComponentByIndex(focusedComponentIdx.value)
+    ? exploreGridRef.value?.getGridComponentRefByIndex(
```

Comment:
Nick you suggested last night that this is what was missing for your "flying context menu" but... diff suggests it was here the whole time??

Reply:
This is just a rename